<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchnote</class>
 <widget class="QWidget" name="searchnote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Suche Notizen</string>
  </property>
  <widget class="QLabel" name="label_searchnote">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Geben Sie ein Suchwort ein:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit_searchnote">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>551</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton_searchnote">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResultsListWidget_searchnote">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>671</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
